docs/content/customization/generic/image-registries.md

```diff
@@ -33,11 +33,71 @@ spec:
       - name: clusterConfig
         value:
           imageRegistries:
-            credentials:
-              - url: https://my-registry.io
+            - url: https://my-registry.io
+              credentials:
                 secretRef:
                   name: my-registry-credentials
 ```
 
 Applying this configuration will result in new files and preKubeadmCommands
 on the `KubeadmControlPlaneTemplate` and `KubeadmConfigTemplate`.
+
+To use a image registry as mirror with CA certificate, specify the following configuration:
+
+If your registry mirror requires self signed CA certifate, create a Kubernetes Secret with keys for `ca.crt`:
```

Comment:
Maybe something like this since its just 1 key?
```suggestion
If your registry mirror requires self signed CA certifate, create a Kubernetes Secret with its contents and key `ca.crt`:
```